lambdatimes/src/components/Content/Content.js

```diff
@@ -18,10 +18,20 @@ export default class Content extends Component {
 
   componentDidMount() {
     // Once the component has mounted, get the data and reflect that data on the state.
+  console.log(this.state.tabs)
+  componentDidMount = () => {
```

Comment:
Notice that this method is nested inside of your componentDidMount on line 19! You want the method to live inside of your class